src/main/main.ts

```diff
@@ -311,37 +317,33 @@ const createWindow = () => {
                     click: () => {
                         main_window.webContents.send('search')
                     },
-                    accelerator: META_KEY + '+shift+f',
+                    accelerator: META_KEY + '+Shift+F',
                 },
                 {
                     label: 'File Search',
                     click: () => {
                         main_window.webContents.send('fileSearch')
                     },
-                    accelerator: META_KEY + '+p',
+                    accelerator: META_KEY + '+P',
                 },
                 {
                     label: 'Command Palette',
                     click: () => {
                         main_window.webContents.send('commandPalette')
                     },
-                    accelerator: META_KEY + '+shift+p',
+                    accelerator: META_KEY + '+Shift+P',
                 },
             ],
         },
     ])
     var menu = Menu.buildFromTemplate(menuList)
     Menu.setApplicationMenu(menu)
 
-    globalShortcut.register(META_KEY + '+=', () => {
```

Comment:
Why delete here?

Reply:
This will cause cursor to listen for the ctrl+plus key in the background, even if cursor is not focused.

There is a problem with overwriting the ctrl+plus key of software such as Chrome.

Instead, bind two local shortcut keys:

```
            label: 'View',
            submenu: [
                {
                    label: 'Zoom In',
                    click: () => {
                        main_window.webContents.send('zoom_in')
                    },
                    accelerator: META_KEY + '+Plus',
                },
                {
                    label: 'Zoom In',
                    click: () => {
                        main_window.webContents.send('zoom_in')
                    },
                    accelerator: META_KEY + '+=',
                },
```